.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "velocitas_cli_devcontainer",
```

Comment:
This is a human readable name shown in the lower left hand corner of VSCode - should be short and shouldn't contain underscores.
`CLI Development` or something similar.